docs/reference/nuget-config-file.md

```diff
@@ -232,6 +235,35 @@ Identifies to the currently active source or indicates the aggregate of all sour
     <add key="All" value="(Aggregate source)" />
 </activePackageSource>
 ```
+## trustedSigners section
+
+Stores trusted signers used to allow package while installing or restoring. This list cannot be empty when the user sets `signatureValidationMode` to `require`. 
```

Comment:
>Stores trusted signers used to allow package while installing or restoring

Does not sounds good to me, I'd rephrase to:

Stores the certificate fingerprint used to evaluate trust  when installing or restoring packages in `require` mode.

Reply:
But don't we want to give the user a sense of authors and repositories instead of just certificates? I like it better to talk about trusted authors and trusted repositories instead of certificates... certificates are just the means of trust, but technically what you want to trust is a given author or a given repository. 

Also, `require` mode by itself doesn't mean much to a new user that doesn't know that `trustedSigners` is related to `signatureValidationMode`